modules/RTC/TraceablePeerConnection.js

```diff
@@ -284,7 +285,7 @@ export default function TraceablePeerConnection(
 
     this.interop = new Interop();
 
-    this.simulcast = new SdpSimulcast({ numOfLayers: SIM_LAYER_RIDS.length });
+    this.simulcast = new SdpSimulcast({ numOfLayers: SIM_LAYERS.length });
```

Comment:
Should we drop the parameter and just get the constant internally in `SdpSimulcast`? It feels like they are very coupled.